srml/contracts/src/lib.rs

```diff
@@ -937,3 +952,62 @@ impl Default for Schedule {
 		}
 	}
 }
+
+/// `SignedExtension` that checks if a transaction would exhausts the block gas limit.
+#[derive(Encode, Decode, Clone, Eq, PartialEq)]
+pub struct CheckBlockGasLimit<T: Trait + Send + Sync>(PhantomData<T>);
+
+#[cfg(feature = "std")]
```

Comment:
```suggestion
```
Please remove, we are going to need it if we want to create transaction in the runtime.